README.md

```diff
@@ -31,6 +31,9 @@ There are two ways to run Joinup. With `docker` and `docker-compose` and buildin
 
 To start with docker, please, check the separated [README file](resources/docker/README.md).
 
+### Mac without docker installation
+To start on a mac and want to do this without docker, please, check the separated [README file](resources/mac/README.md).
```

Comment:
..To start on a mac without docker, please, check the separated..

Reply:
here it says "...To start on a mac without docker, please, check the separated README file." with README file clickable

Reply:
my suggestion was to remove _**and want to do this**_ from the sentence.